tests/testsuite/cargo_remove/optional_dep_feature/out/Cargo.toml

```
@@ -0,0 +1,23 @@
+[package]
+name = "cargo-remove-test-fixture"
+version = "0.1.0"
+
+[[bin]]
+name = "main"
+path = "src/main.rs"
+
+[build-dependencies]
+semver = "0.1.0"
+
+[dependencies]
+docopt = "0.6"
+rustc-serialize = "0.4"
+semver = "0.1"
+toml = "0.1"
+clippy = "0.4"
+
+[dev-dependencies]
+regex = "0.1.1"
+
+[features]
+std = [ "semver/std"]
```

Oops. This space looks not good 😥.

Any way to trim it?

```suggestion
std = ["semver/std"]
```